international-labour-organization.csl

```diff
@@ -158,12 +160,12 @@
             </group>
           </group>
         </else-if>
-        <else-if type="chapter paper-conference" match="any">
-          <text macro="title" prefix=" " suffix="."/>
+        <else-if type="chapter" match="any">
+          <text macro="title" prefix=" &quot;" suffix="&quot;,"/>
```

Comment:
Thanks for the validation. Just back from the holidays and heard there are
also some other changes needed.
I'll update the pull request with "quotes" + other modifications asap.

On Tue, Jul 28, 2015 at 4:09 PM, Rintze M. Zelle <notifications@github.com>
wrote:

> In international-labour-organization.csl
> <https://github.com/citation-style-language/styles/pull/1620#discussion_r35651226>
> :
>
> > @@ -158,12 +160,12 @@
> >              </group>
> >            </group>
> >          </else-if>
> > -        <else-if type="chapter paper-conference" match="any">
> > -          <text macro="title" prefix=" " suffix="."/>
> > +        <else-if type="chapter" match="any">
> > +          <text macro="title" prefix=" &quot;" suffix="&quot;,"/>
>
> It's better to use the "quotes" attribute here (quotes="true"). See
> http://docs.citationstyles.org/en/stable/specification.html#quotes
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1620/files#r35651226>
> .
>